src/themes/default/elements/flag.variables

```diff
@@ -396,7 +396,7 @@
     @1f1e8-1f1ff: {
         countrycode: cz;
         class: czechia;
-        aliasClass: false;
+        aliasClass: czech;
```

Comment:
I wonder what is the motivation and consistency behind this change. `Czech` itself is a language and for other countries there is nothing like `french` class.

Shouldn't this be `czech_republic` (consistent with `united_states` for example)?

Reply:
@mvorisek 
I trust in your decision here as you are a czech native speaker 🙂 , so if you think this is wrong i am fine by reverting or changing the alias name as you think it would be correct.
And your reasons are valid. Sorry for merging this too quick
So what should be do? Revert or Rename?

Reply:
In general I would prefer the data from some external/consistent/popular source, but if the data are managed by Fomantic-UI, it should be renamed to `czech_republic`. Czechia is short form of Czech Republic introduced in 2006 - https://kafkadesk.org/2019/04/19/czech-republic-vs-czechia-a-linguistic-riddle/ but the long form is still used a lot.

Reply:
Fixed by  #2746 and docs by https://github.com/fomantic/Fomantic-UI-Docs/pull/453